src/core/internationalization.js

```diff
@@ -149,8 +152,7 @@ export const initialize = () => {
       },
       backend: {
         fallback: 'en',
-        loadPath:
-          'https://cdn.jsdelivr.net/npm/p5/translations/{{lng}}/{{ns}}.json'
+        loadPath: latestMinorVersionPath + '/translations/{{lng}}/{{ns}}.json'
```

Comment:
and then here,  we could compose it like
```js
loadPath: `${getCDNPath}/translations/{{lng}}/{{ns}}.json`
```

Reply:
and then I think with string interpolation we could simply write  with one liner change:
```js
loadPath: `https://cdn.jsdelivr.net/npm/p5@${constants.VERSION.replace(/^(\d+\.\d+)\.\d+.*$/, '$1')}/translations/{{lng}}/{{ns}}.json`
```